internal/service/alarms/internal/alarms_server.go

```diff
@@ -159,15 +160,19 @@ func (a *AlarmsServer) GetSubscription(ctx context.Context, request api.GetSubsc
 	return api.GetSubscription200JSONResponse(models.ConvertSubscriptionModelToApi(*record)), nil
 }
 
-func (a *AlarmsServer) GetAlarms(ctx context.Context, request api.GetAlarmsRequestObject) (api.GetAlarmsResponseObject, error) {
-	// TODO implement me
+// GetAlarms handles an API request to fetch Alarm Event Records
+func (a *AlarmsServer) GetAlarms(ctx context.Context, _ api.GetAlarmsRequestObject) (api.GetAlarmsResponseObject, error) {
+	records, err := a.AlarmsRepository.GetAlarmEventRecords(ctx)
```

Comment:
Though filter support is currently not there, please log the input and log so that user is aware of this and that we are simply returning all the data .

Reply:
This is how we've handling list for alarm and resource endpoints. If we are gonna modify something, let's do it for all at once in a different pr. Here, I'm following the same format.

Reply:
Filtering is coming in PR #390 to get us back to parity with how the API works in the current/old implementation.  The filtering is not optimal as it looks at the data after it has already been fetched from the db, but we can rework this as a later step... with a goal of getting more functionality in place before we circle back and rework too many things.

Logging input can be handled when we rework the filtering at a later time... and/or we can throw an additional debug log in PR #390